packages/framework/src/Model/Advert/Exception/AdvertPositionNotKnownException.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Shopsys\FrameworkBundle\Model\Advert\Exception;
+
+use Exception;
+
+class AdvertPositionNotKnownException extends Exception implements AdvertException
+{
+    public function __construct(string $positionName, array $knownPositionsNames)
```

Comment:
In Exceptions' constructors, we always add a optional last parameter `Exception $previous = null`. In the past, this was (still is?) a part of the coding standards.

Reply:
True, fixed :+1: